docs/en/observability/whats-new.asciidoc

```diff
@@ -3,15 +3,109 @@
 
 Here are the highlights of what's new and improved in {minor-version}.
 
-coming[8.0.0]
-
-// Other versions:
-// {observability-guide-all}/8.1/whats-new.html[8.1] |
-// {observability-guide-all}/8.0/whats-new.html[8.0] |
-// {observability-guide-all}/7.17/whats-new.html[7.17] |
+Other versions:
+{observability-guide-all}/8.4/whats-new.html[8.3] |
```

Comment:
```suggestion
{observability-guide-all}/8.4/whats-new.html[8.4] |
```

Reply:
Ha, you're too good - I spotted that and fixed it locally but didn't think anyone would notice before I updated!

(I probably should have taken obs-docs off review until PMs had finished reviewing...)